android/app/src/main/java/com/example/learningplatform/ContentSearchFragment.java

```diff
@@ -167,10 +185,24 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
 
     public int exerciseID;
     public String firstCharactersOfExercise;
+    public String exerciseBody;
     public String tagList;
-
-    public ContentDataClass(int id, String exercise, String[] tagList){
+    public String[] options;
+    public String answer;
+
+    public ContentDataClass(int id, String exercise, String[] tagList, String[] options, int answer){
+        if(answer == 1){
+            this.answer = options[0];
+        }else if(answer == 2){
+            this.answer = options[1];
+        }else if(answer == 3){
+            this.answer = options[2];
+        }else if(answer ==4){
+            this.answer = options[3];
+        }
+        this.options = options;
         this.exerciseID = id;
+        this.exerciseBody = exercise;
         if(exercise.length()>100){
             exercise = exercise.substring(0,100);
```

Comment:
Question body length can be 1000

Reply:
serachte exerciseın tamamı çıkmasın ilk 100 karakter gösterilsin sadece diye düşünerek yaptı